### DISCUSSION TITLE
Split files larger than 2Gb

### DISCUSSION THREAD

**Discussion by Sidnemon** [Nekmo/telegram-upload#93] [Ideas]
Divide files larger than 2Gb into 2Gb parts, send them and delete the created parts. If it were possible to carry out the operation "on the fly" it would be perfect.

Thanks.

**Comment by Nekmo** [comment]
Hi @Sidnemon take a look at the `--large-files` option using the `split` option, it does just that.

https://docs.nekmo.org/telegram-upload/usage.html#cmdoption-telegram-upload-large-files

Regards!

**Reply by Sidnemon** [reply]
Thanks for your quick response!
It works perfectly. Your application is very complete.
Thanks! :D

**Comment by kairin** [comment]
Hi @Nekmo 

How about for current premium users, we can upload file up to 4Gb.

can I simply select files that are 4Gb, or am I still limited to 2Gb even though my account is premium user?
